features/comments_and_kudos/comment_moderation.feature

```diff
@@ -307,3 +307,25 @@ Feature: Comment Moderation
     Then I should see "All moderated comments approved."
     When I view the work "Moderation"
     Then I should see "Comments (4)"
+
+  Scenario: I can view the parent thread of an unreviewed comment
+    Given the moderated work "Moderation" by "author" with the approved comment "Test comment" by "commenter"
+      And I am logged in as "new_commenter"
+    When I view the work "Moderation"
+      And I follow "Comments (1)"
+      And I follow "Reply" within ".odd"
+      And I fill in "Comment" with "A moderated reply" within ".odd"
+      And I press "Comment" within ".odd"
+    When I am logged in as "author"
+      And I view the work "Moderation"
+      And I follow "Unreviewed Comments (1)"
+      And I follow "Parent Thread"
+    Then I should see "Test comment"
+    When I view the unreviewed comments page for "Moderation"
+      And I press "Approve"
+    When I am logged in as "new_commenter"
+      And I post the comment "Zero-depth comment" on the work "Moderation"
+    When I am logged in as "author"
+      And I view the work "Moderation"
+      And I follow "Unreviewed Comments (1)"
+    Then I should not see "Parent Thread"
```

Comment:
Could you add a trailing line here? For a few reasons (https://stackoverflow.com/a/5813359 and a few of the other answers there), it's better to have it than not.  If you use VS Code, that can do it for you automatically: https://stackoverflow.com/a/44704969